### PR TITLE
MARVP-27 fix_animation_issue

### PR DESCRIPTION
**Pull request by vijaysamanthapudi**

### CODE REVIEW

**Review by vigneshwarankarunanithi — APPROVED** [review]

File: controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 1]
```diff
@@ -946,7 +948,8 @@ define({
         }
     },
   onClickOfSubmit:function(){
-    this.view.flxRatingContainer.animate(this.createAnimationObject("0dp"), this. getPlatformSpecific(), null);
+    //this.view.flxRatingContainer.animate(this.createAnimationObject("0dp"), this. getPlatformSpecific(), null);
```
Comment (krishk95) [inline controllers/mobile/frmAgendaController/frmAgendaController.js]: Remove Commented Code

File: controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 2]
```diff
@@ -870,7 +871,8 @@ define({
                 if (speakerObject.master_speaker_id == speakers_master[index].speaker_id) {
```
Comment (krishk95) [inline controllers/mobile/frmAgendaController/frmAgendaController.js:875]: Please address the formatting issues wherever applicable

File: userwidgets/tiles.ratingTile/modules/require/ratingTileController.js [hunk 3]
```diff
@@ -71,6 +71,11 @@ define(function() {
       });
         return animationObejct;
       },
+      /**
+         *	@function getPlatformSpecific
+         * 	@description This generic function is to return the animationConfig
+         * 	@private
+         */
       getPlatformSpecific:function(){
```
Comment (krishk95) [inline userwidgets/tiles.ratingTile/modules/require/ratingTileController.js:79]: The function name and functionality is contradicting. Please change the function name according to the task it is performing

File: userwidgets/tiles.ratingTile/modules/require/ratingTileController.js [hunk 4]
```diff
@@ -71,6 +71,11 @@ define(function() {
       });
         return animationObejct;
```
Comment (krishk95) [inline userwidgets/tiles.ratingTile/modules/require/ratingTileController.js:72]: Check the spelling

File: controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 5]
```diff
@@ -876,14 +876,16 @@ define({
...
+                  	this.view["ratingTile" + speakerIndex].setDefaultSelectedIndex();
                 }
             }
         }
         for (speakerIndex; speakerIndex < 3; speakerIndex++) {
             this.view["flxSpeaker" + speakerIndex].isVisible = false;
             this.view["ratingTile" + speakerIndex].isVisible = false;
         }
-    },
+        this.view["ratingTile"].setDefaultSelectedIndex();
```
Comment (krishk95) [inline controllers/mobile/frmAgendaController/frmAgendaController.js:891]: Please check the alignment of the label "Presentations" in Agenda detail

File: controllers/mobile/frmAgendaController/frmAgendaController.js [hunk 6]
```diff
@@ -846,6 +846,7 @@ define({
      * 	@private
      */
     setSpeakerProfile: function(eventObject) {
+      	this.view.flxRatingContainer.isVisible=true;
```
Comment (krishk95) [inline controllers/mobile/frmAgendaController/frmAgendaController.js]: Validate the below flow.

1.Navigate to a particular Agenda Detail
2.Submit the Feedback
3.Navigate to Agenda List
4.Open Agenda Detail of some other agenda

It is directly scrolling to the position where the feedback is present


Related to : https://konysolutions.atlassian.net/browse/MARVP-50

**Review by krishk95 — CHANGES_REQUESTED** [review]
Please Address the PR Comments

File: controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 1 above)
Comment (vijaysamanthapudi) [reply controllers/mobile/frmAgendaController/frmAgendaController.js]: Removed

File: userwidgets/tiles.ratingTile/modules/require/ratingTileController.js (hunk 3 above)
Comment (vijaysamanthapudi) [reply userwidgets/tiles.ratingTile/modules/require/ratingTileController.js:79]: done

File: controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 5 above)
Comment (vijaysamanthapudi) [reply controllers/mobile/frmAgendaController/frmAgendaController.js:891]: Done

File: controllers/mobile/frmAgendaController/frmAgendaController.js (hunk 6 above)
Comment (vijaysamanthapudi) [reply controllers/mobile/frmAgendaController/frmAgendaController.js]: handled animation

**Pull request merged**